packages/arb-token-bridge-ui/src/components/TransferPanel/useTransferReadinessUtils.ts

```diff
@@ -18,9 +25,17 @@ export function getInsufficientFundsErrorMessage({
 
 export function getInsufficientFundsForGasFeesErrorMessage({
   asset,
-  chain
-}: GetInsufficientFundsErrorMessageParams) {
-  return `Insufficient ${asset} to pay for gas fees. Please add more funds to ${chain}.`
+  chain,
+  balance,
+  requiredBalance
+}: GetInsufficientFundsForGasFeesErrorMessageParams) {
+  if (balance === requiredBalance) {
+    // if our formatAmount method rounds these to the same amount, we add a little buffer to the required balance
+    // TODO: this may need to be rescaled for non-18 decimal chains
```

Comment:
Also started non-18 decimals todos on the bridge, added this here: https://www.notion.so/arbitrum/Non-18-Decimals-cd7e16cb9920454a93512bb728d8af28

Reply:
Not a big fan of this though, should we rather have something more generic like "you need more than requiredBalance" to cover this scenario?